### PR TITLE
[RFR] prepare v0.6.0 release

### PR DESCRIPTION
**Pull request by nicolasmure**

### CODE REVIEW

**Comment by deguif** [comment]
Any news on this one?
Will it be merged in a near future or does it need more work?

**Comment by spointecker** [comment]
+1

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+v0.5.3
+======
+
+- Fix composer.json validity #184
+- Be able to install Gaufrette v0.6 #185
+
+v0.5.2
+======
+
+- Declare command as service to fix sf3.4 deprecation #183<Paste>
```
Comment (NiR-) [inline CHANGELOG.md]: Could you remove `<Paste>` please?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
+v0.6.0
+======
+
+In #204 :
```
Comment (NiR-) [inline CHANGELOG.md]: Could you use the same format as for other versions (PR number at the end of each line) please?

File: CHANGELOG.md (hunk 1 above)
Comment (nicolasmure) [reply CHANGELOG.md]: thanks for the review, changes are done ;)

**Comment by deguif** [comment]
Is there anything I can do to help releasing a new version (some last minutes tasks)?
Maybe we can also add https://github.com/KnpLabs/KnpGaufretteBundle/pull/211 but there are some code styles changes that pollute.

**Comment by nicolasmure** [comment]
I'm going to release `v0.6.0` as it as it should have been released since a while. The PRs you mentioned will land in the next release.
BTW, we're sorry for the lack of activity on the OSS projects these times, we were busy with other projects.
We still appreciate very much the contributions and the efforts you guys are making here :) Thank you :)

**Pull request merged**